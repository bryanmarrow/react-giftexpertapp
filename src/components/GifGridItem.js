import React from 'react'
import PropTypes from 'prop-types';

export const GifGridItem = ( { id, title, url} ) => {

    console.log(id, title, url);

    return (
        <div className="card animate__fadeIn">
            <img src={ url } alt={ title } />
            <p>{ title }</p>
        </div>
    )
}

GifGridItem.PropTypes = {
    title: PropTypes.string.isRequired,
    url: PropTypes.string.isRequired 
}
