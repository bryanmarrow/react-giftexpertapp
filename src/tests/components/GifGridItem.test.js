import React from 'react'
import { shallow } from "enzyme"
import { GifGridItem } from "../../components/GifGridItem"

describe('Pruebas en <GifGridItem />', () => {


    test('debe mostrar correctamente ', () => {

        const wrapper = shallow( <GifGridItem /> )
        expect(wrapper).toMatchSnapshot();
        
    })
    

})