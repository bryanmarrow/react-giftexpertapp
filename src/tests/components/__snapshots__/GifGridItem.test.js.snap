// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifGridItem /> debe mostrar correctamente  1`] = `
<div
  className="card animate__fadeIn"
>
  <img />
  <p />
</div>
`;
